docs/Troubleshooting.md

```diff
@@ -37,8 +37,8 @@ localhost                  : ok=52   changed=12   unreachable=0    failed=1    s
   ```
 
 ### Diagnosis:
-A release of sassoftware/viya4-monitoring-kubernetes prior to 1.2.0 was run by a release of sassoftware/viya4-deployment at release 4.13.0 or later.
-Releases of sassoftware/viya4-monitoring-kubernetes prior to 1.2.0 do not support the installation of OpenSearch logging software which sassoftware/viya4-deployment 4.13.0 or later will attempt to install.
+The cluster-logging task of sassoftware/viya4-monitoring tried to execute code from a release of sassoftware/viya4-monitoring-kubernetes prior to 1.2.0 using a release of sassoftware/viya4-deployment at release 5.0.0 or later.
+Releases of sassoftware/viya4-monitoring-kubernetes prior to 1.2.0 do not support the installation of the OpenSearch logging software which sassoftware/viya4-deployment 5.0.0 or later installs.
```

Comment:
I think I would recast things a bit.  Maybe something like:  
_Release 5.0.0 (and later) of sassoftware/viya4-deployment is only compatible with sassoftware/viya4-monitoring-kubernetes  release 1.2.0. (and later)._

Or, if you feel that OpenSearch must be mentioned, maybe something like:
_Release 1.2.0 (and later) of sassoftware/viya4-monitoring-kubernetes use OpenSearch (rather than Open Distro for Elasticsearch) as search back-end for log monitoring.  Therefore, release 5.0.0 (and later) of sassoftware/viya4-deployment is only compatible with sassoftware/viya4-monitoring-kubernetes  release 1.2.0. (and later)._

Reply:
suggestions, applied